canonical-websites/www.ubuntu.com
# Add CircleCI workflow

## Done

Add CircleCI workflow to the project. Steps added are:
- Python tests.
- Python lint
- Sass lint
- Linkchecker

## QA
Check the CI wokrflow under my CircleCI space.

OR

- Fork www.ubuntu.com.
- Add this feature branch to your fork.
- Enable project in your own CircleCI space.
- Follow the steps in https://circleci.com/docs/2.0/examples/#section=configuration to trigger a build.
- Check the workflow ran properly.

## Issue / Card
Fixes https://github.com/ubuntudesign/base-squad/issues/211
